api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -39,9 +39,24 @@
  * methods will be written to the database at startup if they don't exist yet.
  */
 public final class OpenmrsConstants {
-
-	private OpenmrsConstants() {
-	}
+	/**
+	 * The directory which OpenMRS should attempt to use as its application data directory
+	 * in case the current users home dir is not writeable (e.g. when using application servers
+	 * like tomcat to deploy OpenMRS).
+	 *
```

Comment:
why are you moving this up? I think this section should stay where it was. try to change only whats necessary to achieve the goal of the issue you are working on

Reply:
@teleivo ok I will make the necessary changes.